contrib/benchmarking_nd4j/src/main/java/org/nd4j/memorypressure/MemoryPressureTest.java

```diff
@@ -10,12 +10,15 @@ public class MemoryPressureTest {
     public static void main(String...args) throws Exception {
         Thread[] threads = new Thread[Runtime.getRuntime().availableProcessors()];
         for (int i = 0; i < threads.length; i++) {
+            INDArray arrOne = Nd4j.ones(200);
+            INDArray arrTwo = Nd4j.ones(200);
             threads[i] = new Thread(() -> {
                 AtomicInteger atomicInteger = new AtomicInteger(0);
                 while (atomicInteger.incrementAndGet() < 100000) {
-                    INDArray arrOne = Nd4j.ones(200);
-                    INDArray arrTwo = Nd4j.ones(200);
+
                     arrOne.addi(arrTwo);
+                    System.out.println("Completed " + atomicInteger.get() + " iterations");
```

Comment:
Is this intentional that you create 100000 prints here?

Reply:
Yes this one runs for a LONG time and the main use case isn't throughput here but GC.